Tasks/PowerShellV2/task.loc.json

```diff
@@ -18,7 +18,7 @@
   "version": {
     "Major": 2,
     "Minor": 190,
```

Comment:
Could you please update version to 2.191.0 instread - since current sprint is 191?